app/code/Magento/Pwa/Test/Mftf/Test/VeniaCartPageEditCardAndCheckout.xml

```diff
@@ -8,35 +8,34 @@
 
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
-    <!--<test name="RemoveItemFromTheCartPage">-->
-        <!--<annotations>-->
-            <!--<features value="PWA"/>-->
-            <!--<stories value="Kebab Features"/>-->
-            <!--<title value="Verify user should be able to Remove an item from the cart"/>-->
-            <!--<description value="Verify user should be able to Remove an item from the cart"/>-->
-            <!--<severity value="CRITICAL"/>-->
-            <!--<testCaseId value="MC-21749"/>-->
-            <!--<group value="PWA"/>-->
-        <!--</annotations>-->
-
-        <!--<before>-->
-            <!--<actionGroup ref="GoToVeniaDesktopStorefront" stepKey="goToVeniaDesktopStorefront1"/>-->
-            <!--<actionGroup ref="GoToProductDetailsPage" stepKey="goToProductDetailsPage">-->
-                <!--<argument name="product" value="ValeriaTwoLayerTank"/>-->
-            <!--</actionGroup>-->
-
-            <!--<actionGroup ref="SetProductColorAndSizeAndQuantityOptions" stepKey="addProductToCartWithColorAndSizeAndQuantityOptions1"/>-->
-
-            <!--<actionGroup ref="TemporaryWait3" stepKey="waitForElementToLoad"/>-->
-
-            <!--<actionGroup ref="AddToCart" stepKey="addToCart1"/>-->
-
-            <!--<actionGroup ref="TemporaryWait5" stepKey="waitForElementToLoad5"/>-->
-
-        <!--</before>-->
-
-        <!--<actionGroup ref="GoToCartPage" stepKey="removeItemFromTheCart"/>-->
-        <!--<actionGroup ref="RemoveItemFromTheCart" stepKey="removeItemFromTheCart"/>-->
-
-    <!--</test>-->
+    <test name="CartPageEditProductAndCheckout">
+        <annotations>
+            <features value="PWA"/>
+            <stories value="Cart Page"/>
+            <title value="Edit product on cart page and checkout"/>
+            <description value="Edit product on cart page and checkout"/>
+            <severity value="CRITICAL"/>
+            <testCaseId value="MC-32298"/>
+            <group value="PWA"/>
+            <group value="Cart"/>
+        </annotations>
+
+        <before>
+            <actionGroup ref="GoToVeniaDesktopStorefront" stepKey="goToVeniaDesktopStorefront1"/>
```

Comment:
Just curious, can we not have an action group to add an item to cart? I have seen this in multiple places.

Reply:
we do have one  <actionGroup ref="AddToCart" stepKey="addToCart1"/>